spring-integration-file/src/main/java/org/springframework/integration/file/FileWritingMessageHandler.java

```diff
@@ -443,6 +447,19 @@ public void setChmod(int chmod) {
 		this.permissions = permissions;
 	}
 
+	/**
+	 * Set the callback to use when creating new files. This callback will only be called
+	 * if {@link #fileExistsMode} is {@link FileExistsMode#APPEND} or {@link FileExistsMode#APPEND_NO_FLUSH}
+	 * and new file has to be created. The callback receives the new result file and the message that
+	 * triggered the handler.
+	 * @since 5.1
```

Comment:
`@since` must go in the end of method Javadoc. Well, before `@deprecate` though, but still.
Here is tag order:
```
<property name="tagOrder" value="@param, @return, @throws, @since, @deprecated, @see"/>
```

And no blank lines in the method Javadocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
My bad